Javascript-2/javascript3.js

```diff
@@ -0,0 +1,26 @@
+console.log("EXERCITIUL 3");
+
+const text="mama";
+const lungimeText=text.length;
+const primaLitera=text.charAt(0);
+console.log(lungimeText);
```

Comment:
avoid leavening unnecessary console.log in the code. They can be used for testing or in case you want to display something that you really need